packages/dhis2/src/Adaptor.js

```diff
@@ -500,31 +452,41 @@ export function update(
  * get('tracker/events');
  * @example <caption>Get the relationship between two tracker entities. The only required parameters are 'trackedEntity', 'enrollment' or 'event'. See [Relationships docs](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-241/tracker.html#relationships-get-apitrackerrelationships)</caption>
  * get('tracker/relationships', {
- *   trackedEntity:['F8yKM85NbxW'],
+ *   query: { trackedEntity:['F8yKM85NbxW'] }
+ * });
+ * @example <caption>Get an image from a trackedEntityInstance.</caption>
+ * get('trackedEntityInstances/qHVDKszQmdx/BqaEWTBG3RB/image', {
+ *   headers:{
+ *       Accept: 'image/*'
+ *   },
+ *   parseAs: 'text',
```

Comment:
Didn't we say we'd support `parseAs: 'base64'` ?

That means we can drop the `asBase64` key completely